tests/unit/lms/models/group_info_test.py

```diff
@@ -16,7 +16,7 @@ def test_persist_and_retrieve(self, db_session, group_info):
         assert (
             retrieved_group_info.authority_provided_id == "test_authority_provided_id"
         )
-        assert retrieved_group_info.consumer_key == "test_consumer_key"
+        assert retrieved_group_info.consumer_key == application_instance.consumer_key
```

Comment:
I really like this style, as long as this is _why_ this value is this value, rather than just correlated. I made that mistake in a few places as the consumer key gets copied about quite a bit.

So not only does the consumer key _happen_ to equal the application instance consumer key, that's _why_ it's set like that.